### DISCUSSION TITLE
v0.6.2

### DISCUSSION THREAD

**Discussion by umonaca** [kikoeru-project/kikoeru-express#63] [General]
### 更新历史

- 修正Webkit内核浏览器无法单曲循环的问题  
- 依赖安全更新

v0.6.0版变化请见GitHub Release上[相应的页面](https://github.com/kikoeru-project/kikoeru-express/releases/tag/v0.6.0)。  

### 使用方法
见[使用说明](https://github.com/umonaca/kikoeru-express/blob/unstable/%E7%94%A8%E6%88%B7%E6%96%87%E6%A1%A3.md)。
简而言之，以下任选其一：
- Windows用户下载`kikoeru-win-x64-v0.6.2.zip`，直接解压即可使用。无需安装node等
- Mac OS用户下载`kikoeru-darwin-x64-v0.6.2.zip`，解压执行其中的`kikoeru-express`，并赋予其执行权限（可在命令行下`chmod +x kikoeru-express`）。遇到权限提示请选择允许
- 请使用Docker Hub上的`muveex/kikoeru:v0.6.2`，不再提供打包镜像。注：`muveex/kikoeru-express`与`muveex/kikoeru`内容相同
- 也可使用[docker-compose](https://github.com/umonaca/kikoeru-express/blob/v0.6.2/docker-compose.yml)。可自行编辑文件调整挂载位置
- 源码运行：从[前端项目页面](https://github.com/umonaca/kikoeru-quasar/releases/tag/v0.6.2)下载`spa-v0.6.2.zip`或`pwa-v0.6.2.zip`（如果不使用PWA或没有HTTPS证书，请使用`spa-v0.6.2.zip`），然后解压到本项目`dist/`文件夹下。或者也可以手动quasar build前端页面，然后把前端项目文件夹下生成的`dist/spa/`的内容整个复制/软连接/挂载/NTFS Junction过来等。请使用`git checkout v0.6.2`切换到此版本的源代码。然后执行`npm install`和`node app.js`。

<hr /><em>This discussion was created from the release <a href='https://github.com/kikoeru-project/kikoeru-express/releases/tag/v0.6.2'>v0.6.2</a>.</em>

**Comment by NaoBao1911** [comment]
Windows版现在8位RJ号的添加不了了 怎么解决啊
